src/js/components/PageHeader/__tests__/PageHeader-test.tsx

```diff
@@ -26,6 +26,24 @@ describe('PageHeader', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  test('size', () => {
+    const { asFragment } = render(
+      <Grommet>
+        <PageHeader
+          title="Grommet"
+          subtitle={`Grommet helps you build responsive and accessible 
+          mobile-first projects for the web with an easy to use component 
+          library.`}
+          actions={<Button label="Get Started" primary />}
+          parent={<Anchor label="Parent Page" />}
+          size="large"
```

Comment:
Can we do all sizes via unit test? That would cover "small" and explicitly setting "medium" as well.

Reply:
Yes, will update.

Reply:
Done.